### PR TITLE
Adds further comments to MC docs

### PR DESCRIPTION
**Pull request by JosephSalisbury**
Towards https://github.com/giantswarm/giantswarm/issues/15700
From https://github.com/giantswarm/docs/pull/836

### Please remember to

- [x] Give the PR a meaningful title -- it will be shown in the release notes
- [x] Add (or remove) [user questions](https://github.com/giantswarm/docs/blob/master/CONTRIBUTING.md#front-matter) associated with any updated docs

### CODE REVIEW

File: src/content/general/management-clusters/index.md [hunk 1]
```diff
@@ -21,13 +21,19 @@ As we are fully convinced of Kubernetes as a platform for building platforms, we
...
 
-We deploy new instances of the operators with this new version, via the [release-operator](https://github.com/giantswarm/release-operator/) to avoid impacting existing workload clusters. These new operators will not become active until existing workload clusters are upgraded or a new cluster is created.
+To make sure that existing workload clusters aren't affected by new versions, our operators are immutable. With each new release, we deploy new instances of of the operators with the new version, via the [release-operator](https://github.com/giantswarm/release-operator/), which avoids impacting existing workload clusters. These new operators will not become active until existing workload clusters are upgraded or a new cluster is created. Overall, this entails that the operators are versioned together with the cloud provider infrastructure and component versions (e.g: Kubernetes, CoreDNS, etc.) of the cluster itself, with the new operators taking over when the cluster is upgraded to the new version.
```
Comment (rossf7) [inline src/content/general/management-clusters/index.md:36]:
```suggestion
To make sure that existing workload clusters aren't affected by new versions, our operators are immutable. With each new release, we deploy new instances of the operators with the new version, via the [release-operator](https://github.com/giantswarm/release-operator/), which avoids impacting existing workload clusters. These new operators will not become active until existing workload clusters are upgraded or a new cluster is created. Overall, this entails that the operators are versioned together with the cloud provider infrastructure and component versions (e.g: Kubernetes, CoreDNS, etc.) of the cluster itself, with the new operators taking over when the cluster is upgraded to the new version.
```

File: src/content/general/management-clusters/index.md [hunk 2]
```diff
@@ -21,13 +21,19 @@ As we are fully convinced of Kubernetes as a platform for building platforms, we
...
+
+We do not support the upstream Cluster API for all our providers yet, but are fully committed to supporting this open standard soon.
+
 ## Workload Cluster releases
 
 Our workload clusters are versioned using [workload cluster releases]({{< relref "/general/releases" >}}). From a workload cluster point of view, upgrades are described in [cluster upgrades]({{< relref "/general/cluster-upgrades" >}}).
 
-When we publish a new workload cluster release, we create a new [Release]({{< relref "/ui-api/management-api/crd/releases.release.giantswarm.io.md" >}}) custom resource and any new operator versions are deployed to the management cluster.
+A core focus of our development in our versioning is to guarantee stability and reliability. Our upgrades follow a lockstep approach, where we move from one tested set of versions of our components to the next set of versions of our components. This strict versioning also allows you to test out new versions, test upgrades, and even integrate cluster creation into your own CI, such as to run integration tests.
```
Comment (rossf7) [inline src/content/general/management-clusters/index.md:32]:
```suggestion
A core focus of development in our versioning approach is to guarantee stability and reliability. Our upgrades follow a lockstep approach, where we move from one tested set of versions of our components to the next set of versions of our components. This strict versioning also allows you to test out new versions, test upgrades, and even integrate cluster creation into your own CI, such as to run integration tests.
```

The two uses of our doesn't flow for me for some reason. What about this? I think our approach is something unique about our offer.

So overall I like this explanation a lot and good to go deeper here. 👍

**Review by rossf7 — APPROVED** [review]
LGTM just a couple of minor things

**Comment by JosephSalisbury** [comment]
thanks @rossf7, that does get it better <3

**Comment by JosephSalisbury** [comment]
get it better? make it better?

i am all out of the english goodness now

**Pull request merged**